docs/extend/modular-resources/configure.md

```diff
@@ -1,30 +1,171 @@
+| `type` | string | **Required** | The {{< glossary_tooltip term_id="subtype" text="subtype">}} of the [API](/extend/modular-resources/key-concepts/#apis) (the third part of the {{< glossary_tooltip term_id="api-namespace-triplet" text="API namespace triplet">}}). |
+| `name` | string | **Required** | What you want to name this instance of your modular resource. |
+| `model` | string | **Required** | The full {{< glossary_tooltip term_id="model-namespace-triplet" text="model namespace triplet">}} of the modular resource's [model](/extend/modular-resources/key-concepts/#models). |
+| `depends_on` | array | Optional | The `name` of components you want to confirm are available on your robot alongside your modular resource. Often a [board](/components/board/). Unnecessary if you coded [implicit dependencies](/extend/modular-resources/key-concepts/#dependency-management). |
+
+All standard properties for configuration, such as `attributes` and `depends_on`, are also supported for modular resources.
+The `attributes` available vary depending on your implementation.
+
+{{< tabs >}}
+{{% tab name="JSON Template" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+  "components": [
+    {
+      "namespace": "<your-module-namespace>",
+      "type": "<your-resource-subtype>",
+      "model": "<model-namespace>:<model-family>:<model-name>",
+      "name": "<your-model-instance-name>",
+      "depends_on": [],
+    }
+  ],
+  "modules": [ ... ] // < INSERT YOUR MODULE CONFIGURATION >
+}
+```
+
+{{% /tab %}}
+{{% tab name="JSON Example" %}}
+
+The following is an example configuration for a base modular resource implementation.
+The configuration adds `acme:demo:mybase` as a modular resource from the module `my_base`.
+The custom model is configured as a component with the name "my-custom-base-1".
+You can send commands to the base according to the Viam [base API](/components/base/#api):
+
+```json {class="line-numbers linkable-line-numbers"}
+{
+    "components": [
+        {
+            "type": "board",
+            "name": "main-board",
+            "model": "pi"
+        },
+        {
+        "type": "base",
+        "name": "my-custom-base-1",
+        "model": "acme:demo:mybase",
+        "namespace": "rdk",
+        "attributes": {},
+        "depends_on": [ "main-board" ]
+        }
+    ],
+    "modules": [
+    {
+      "name": "my-custom-base",
+      "executable_path": "/home/my_username/my_base/run.sh"
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{% /tabs %}}
+
+## Add a local module to your robot
+
+If you are developing your own modular resource, and intend to deploy it to your robot locally, first follow [these steps](/extend/modular-resources/create/) to code your own module and generate an executable.
+If you are using a pre-built modular resource, make sure you install the module and determine the filename of [the module's executable](/extend/modular-resources/create/#compile-the-module-into-an-executable).
```

Comment:
Why would someone use a pre-built one locally? I wonder if it'd make more sense to introduce this use case in a separate paragraph here? Also what exactly is meant by pre-built? Is it something from the registry, but downloaded independently? I also fear that someone might confuse "pre-built" with "built-in" and get real confused :D

Reply:
This is following shortly, separately.